pylint/checkers/stdlib.py

```diff
@@ -105,6 +107,12 @@ class StdlibChecker(BaseChecker):
                   'The warning is emitted when a threading.Thread class '
                   'is instantiated without the target function being passed. '
                   'By default, the first parameter is the group param, not the target param. '),
+        'W1507': ('Using copy.copy(os.environ). Use os.environ.copy() '  # TODO: number
```

Comment:
Please check if I chose the right number for this warning

Reply:
Yup, it's fine as is!

Reply:
Then #TODO should be remove